AnnotatoBackend/Sources/App/WebSocketControllers/OfflineToOnlineWebSocketController.swift

```diff
@@ -0,0 +1,90 @@
+import Foundation
+import Vapor
+import AnnotatoSharedLibrary
+import FluentKit
+
+class OfflineToOnlineWebSocketController {
+    private static var logger = Logger(label: "OfflineToOnlineWebSocketController")
+
+    static func handleOfflineToOnlineResolution(userId: String, data: Data, db: Database) async {
+        do {
+            Self.logger.info("Processing offline to online data...")
+
+            let message = try JSONCustomDecoder().decode(AnnotatoOfflineToOnlineMessage.self, from: data)
+
+            switch message.mergeStrategy {
+            case .keepServerVersion:
+                await handleKeepServerVersion(userId: userId, db: db, message: message)
+            case .overrideServerVersion:
+                await Self.handleOverrideServerVersion(userId: userId, db: db, message: message)
+            }
+        } catch {
+            Self.logger.error("Error when handling incoming offline to online data. \(error.localizedDescription)")
+        }
+    }
+
+    private static func handleKeepServerVersion(
```

Comment:
After the user goes offline, from that point onwards, documents and annotations might be created/updated/deleted in the server by other users. 

Since we want to keep the server's version, do we have to send all these documents that have been created/updated/deleted after the last online time so that the user can update their local version?

Reply:
My bad. Didn't know that create and delete actually updates `updatedAt`.